package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "build:ci": "ncc build src/adapters/github/github-actions.ts -o ./",
     "build:serverless": "ncc build src/index.ts -o ./",
-    "build": "tsc",
+    "build": "tsc && copyfiles src/assets/images/* lib/",
```

Comment:
Maybe cleaner to make a new `postbuild` script. 

https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts